src/sparql/get-terms-by-regex.sparql.jinja2

```diff
@@ -0,0 +1,9 @@
+prefix owl:  <http://www.w3.org/2002/07/owl#>
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+
+select ?term_id ?term_label where {
+  ?term_id a owl:Class;
+    skos:prefLabel ?term_label .
```

Comment:
Almost none of the ontologies we care about use skos:prefLabel. most use `rdfs:label`. If you want to obtain the label this way you will have to use VALUES { rdfs:label skos:prefLabel }

Reply:
Hmmm, I don't understand the need for `VALUES` here but no need to worry about explaining if you're short on time. Idk why I included return of labels here. For this one, I might have thought it was necessary to select the label in order for the regex to work. But I'm removing it now and I imagine it'll run fine.